internal/common/simulation/ode_test.go

```diff
@@ -0,0 +1,127 @@
+package simulation
+
+import (
+	"fmt"
+	"math"
+	"testing"
+)
+
+// helper type to group f(x) and f'(x) for various f(x)
+type testFuncPair struct {
+	y     yActual
+	dydt  ypFunc
+	label string
+}
+
+var testPairs = []testFuncPair{
```

Comment:
```suggestion
var testPairs = [...]testFuncPair{
```

Reply:
in the event we run parallel tests (probably never) making it a literal helps prevent accidental mutation of this